opg-base-1604/Dockerfile

```diff
@@ -34,13 +34,27 @@ RUN echo "Updating Packages" &&\
                        zip \
                        netcat \
                        netcat-traditional \
-                       iproute2 sudo &&\
+                       iproute2 \
+                       sudo &&\
```

Comment:
👎 I'm really against adding `sudo` into more (basically all) containers.

Reply:
As per our slack conversation, this is fine for now. As we need to have this for migrations down the road. 

Adding `sudo` to containers opens up security problems. If we're adding sudo we might as well do away with the user and run as `root` which simplifies everything and doesn't pretend to give us any protection.

Reply:
@damacus yes I agree. sudo is required for ingest. Can you raise a ticket and I can have this put into Sirius upcoming sprints?